corehq/apps/users/static/users/js/roles.js

```diff
@@ -1,5 +1,6 @@
 hqDefine('users/js/roles',[
     'knockout',
+    'jquery',
```

Comment:
You need to also add `$` to the parameters list. It works as is because we're currently in a weird state thats' half-migrated to requirejs, but the ideal future state is all dependencies are explicitly declared.